testsuite/run_sets/build_validation_add_common_channels.yml

```diff
@@ -1,17 +1,13 @@
-# AUTOGENERATED using : rake utils:generate_add_custom_repositories_features
-
 ## Add common channels and special repositories BEGIN ###
 
-- features/build_validation/add_custom_repositories/add_ubuntu_repositories.feature
-- features/build_validation/add_custom_repositories/srv_prepare_clm_filters.feature
-- features/build_validation/add_custom_repositories/add_alma9_repositories.feature
-- features/build_validation/add_custom_repositories/add_centos7_repositories.feature
-- features/build_validation/add_custom_repositories/add_liberty9_repositories.feature
-- features/build_validation/add_custom_repositories/add_oracle9_repositories.feature
-- features/build_validation/add_custom_repositories/add_rocky8_repositories.feature
-- features/build_validation/add_custom_repositories/add_rocky9_repositories.feature
+- features/build_validation/add_common_repositories/ubuntu1804_minion_add_common_repositories.feature
```

Comment:
No, those are not common, but custom channels that has been CLM filtered initially.
I suggest you a different stage for CLM filters and a different run_set then.

Reply:
That's indeed a solution. But that would leave only the Ubuntu feature in the "common channels". It would feel a bit alone :smile_cat: .

What about finding a common name for Ubuntu and the CLM-filtered features, given they are in a common bag  in the pipelines and controlled by the same build parameter?

Reply:
I would not re-use the word "common" for different scopes:
SUMA scope
Test scope

That will cause missunderstandings.

Reply:
Absolutely agree. And it puzzled me for a long time in our pipelines where we have
```
add_custom_channels --> for MUs
add_common channels --> for common channels and CLM
```
That was both obscure and misleading.